packages/contracts/src/ubiquistick/SimpleBond.sol

```diff
@@ -183,6 +186,7 @@ contract SimpleBond is ISimpleBond, Ownable, Pausable {
 
             assert(bnd.claimed <= bnd.rewards);
```

Comment:
update the state var here, and in 180, too https://github.com/ubiquity/ubiquity-dollar/pull/714/files#diff-39a1832a63be8f745df33562f0b2f0a49b3e69356f50e357e05de5a1e47d5feeR180

Reply:
You mean renaming `bnd` to `bondState` >

Reply:
to enhance readability, for further uses, if we're going to keep the contract

Reply:
good reability == good maintanibility find akward this vars so short this way